Bottom: b5ba01a6f86e9daad8f40e783f9e7df6d63a19fb
Top:    082a42ab4ab3e212dba561497e15cf70ee3cd5f3
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-23 17:25:15 +0000

Refresh of proposed-fixes

---

diff --git a/src/report.c b/src/report.c
index 7d9063a..da4b600 100644
--- a/src/report.c
+++ b/src/report.c
@@ -688,8 +688,8 @@ ReportAllNetLengths (int argc, char **argv, int x, int y)
       continue;
 
     got_one:
-      ResetConnections (true, true);
-      /* FIXME: Need a serial number increment?? */
+      if (ResetConnections (true, true))
+        IncrementUndoSerialNumber ();
       length = XYtoNetLength (x, y, &found);
 
       gui->log("Net %s length %.*f %s\n", netname, prec, length*scale, units_name);
@@ -704,8 +704,8 @@ ReportNetLength (int argc, char **argv, int x, int y)
   char *netname = 0;
   int found = 0;
 
-  ResetConnections (true, true);
-  /* FIXME: Need a serial number increment?? */
+  if (ResetConnections (true, true))
+    IncrementUndoSerialNumber ();
   gui->get_coords ("Click on a connection", &x, &y);
 
   length = XYtoNetLength (x, y, &found);
